cypress/integration/pages/article/amp.js

```diff
@@ -0,0 +1,134 @@
+import config from '../../../support/config/services';
+import appConfig from '../../../../src/app/lib/config/services';
+
+const serviceHasArticlePageType = service =>
+  config[service].pageTypes.articles !== undefined;
+
+Object.keys(config)
+  .filter(serviceHasArticlePageType)
+  .forEach(service => {
+    describe(`Article - Amp - ${service}`, () => {
+      before(() => {
+        cy.visit(`${config[service].pageTypes.articles}.amp`);
+      });
+
+      it('Metadata', () => {
+        cy.request(`${config[service].pageTypes.articles}.json`).then(
+          ({ body }) => {
+            cy.get('meta[name="description"]').should(
+              'have.attr',
+              'content',
+              body.promo.summary || body.promo.headlines.seoHeadline,
+            );
+            cy.get('meta[name="og:title"]').should(
+              'have.attr',
+              'content',
+              body.promo.headlines.seoHeadline,
+            );
+            cy.get('meta[name="og:type"]').should(
+              'have.attr',
+              'content',
+              body.metadata.type,
+            );
+            cy.get('meta[name="article:published_time"]').should(
+              'have.attr',
+              'content',
+              new Date(body.metadata.firstPublished).toISOString(),
+            );
+            cy.get('meta[name="article:modified_time"]').should(
+              'have.attr',
+              'content',
+              new Date(body.metadata.lastPublished).toISOString(),
+            );
+
+            cy.get('html').should(
+              'have.attr',
+              'lang',
+              body.metadata.passport.language,
+            );
+          },
+        );
+      });
+
+      it('should have AMP attribute', () => {
+        cy.get('html').should('have.attr', 'amp');
+      });
+
+      it('should have lang and dir attributes', () => {
+        cy.request(`${config[service].pageTypes.articles}.json`).then(
+          ({ body }) => {
+            cy.hasHtmlLangDirAttributes({
+              lang: body.metadata.passport.language,
+              dir: appConfig[service].dir,
+            });
+          },
+        );
+      });
+
+      it('should load the AMP framework', () => {
```

Comment:
This doesn't seem right, we're testing it's there but not that it's loaded. So it's not really an e2e. Then again if we test it's loaded we are dependent on it existing and that's not up to us.

For now at least comment a TODO to refactor or review